src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -647,7 +623,8 @@ protected Object readResolve() {
                     updateJiraIssueForAllStatus,
                     groupVisibility,
                     roleVisibility,
-                    useHTTPAuth);
+                    useHTTPAuth,
```

Comment:
@olamy I think this whole `if` migration block is there for a long time as well, everyone should have already migrated those credentials so I propose to drop this whole `if` as well?